public/css/layout.css

```diff
@@ -16,7 +16,7 @@ body {
 
  /*  General and Miscellaneous Tag Styles */
 label, input {
-    float: left;
+    float: right;
```

Comment:
You are moving all controls to the left, create a css directive that is more focused to your case. Th is affecting all labels and inputs.